### ISSUE TITLE
Make search input more visible

### ISSUE THREAD

**Issue by Feridun12** [Feridun12/React-technical-testing#20]
While implementing the new fonts for the search input and search button, I have realised that the input is not visible when hovered or when it is active. That is not good for accessibility.

<img width="605" alt="image" src="https://github.com/Feridun12/React-technical-testing/assets/89600036/0e8a4ee2-9180-4d8e-9936-e4bdfd3b9007">
